docs/advanced-usage/access-control/voms-proxy/index.md

```diff
@@ -0,0 +1,65 @@
+# Voms-proxy
+
+If your workflow needs special permissions that your Virtual Organization entitles you to, the VOMS client can create a proxy certificate for your authentication.
+
+## Testing your credentials
+
+Before moving to REANA, check that your credentials are in order.
+
+```console
+# Login to lxplus
+$ ssh johndoe@lxplus.cern.ch
+# Make sure userkey.pem is read/write only by the owner
+$ ls -l ~/.globus
+-rw-r--r--. usercert.pem
+-r--------. userkey.pem
+
+# Let us create a proxy certificate with the VO cms
+$ voms-proxy-init --voms cms
+Enter GRID pass phrase for this identity:
+
+Contacting voms2.cern.ch:15002 [/DC=ch/DC=cern/OU=computers/CN=voms2.cern.ch] "cms"...
+Remote VOMS server contacted succesfully.
+
+
+Created proxy in /tmp/x509up_u131816.
+
+Your proxy is valid until Wed Apr 01 00:43:08 CEST 2020
+```
+
+## Uploading secrets
+
+To create the proxy certificate the VOMS client needs access to your user certificate `usercert.pem`, encrypted private key `userkey.pem` and GRID pass phrase. For REANA the pass phrase needs to be encoded in Base64. To encode it exchange `mygridpassphrase` for your GRID pass phrase and perform
+
+```console
+$ echo 'mygridpassphrase' | base64
+bXlncmlkcGFzc3BocmFzZQo=
+```
+
+Upload these three secrets to REANA. In addition the client needs information about your virtual organization. This is communicated by the environment variable `VOMSPROXY_VO` and can also be added to the secrets:
+
+```console
+$ reana-client secrets-add --file userkey.pem \
+                           --file usercert.pem \
+                           --env VOMSPROXY_PASS=bXlncmlkcGFzc3BocmFzZQo= \
+                           --env VOMSPROXY_VO=cms
+```
+
+## Setting voms-proxy requirement
+
+Set `voms_proxy: true` for the steps that need a proxy certificate in the workflow specification.
+Please note that step's docker image (e.g ``environment: 'reanahub/reana-auth-vomsproxy'``)
+should have the VOMS client installed for the command `voms-proxy-info` to work.
```

Comment:
Here we say that they should have the VOMS client installed in their image, which makes sense... if I would have started reading from here I wouldn't have made any of the previous comments. 

Why my confusion? I think that if they follow these docs they should take for granted that everything is going to work out of the box without having to bother about the details, that's why I would always make reference to REANA as a whole. Although that would be ideal, we have to tell them that the VOMS client has to be installed... so they have to think of how it works, so it is not really "out of the box" (same happens with Kerberos), so all the explanations from before would make sense.

Reply:
I will remove the mentions of the VOMS client